recipes/scikit-multiflow/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "scikit-multiflow" %}
+{% set version = "0.1.4" %}
+{% set sha256 = "69fa62479d6b4cf3aa04800fde3c1cf012c3350fff1cc6f5648a0e646ecbfc5d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
+  script:
+    - "{{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt"
```

Comment:
```suggestion
    - "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Reply:
Changed in f31ce0c380863ab695530ba37e0b80ed3aa3ec5d

Reply:
Added pytest and pytest-runner in f67502d11553170392bb0f9e81aa1bc6ab0dc6a5 toavoid RunTimeError due to setuptools